### ISSUE TITLE
Error during installation AFTER 500 EVR payment to registry

### ISSUE THREAD

**Issue by jmos1277** [EvernodeXRPL/evernode-host#180]
How can I get my 500 Evers back?
The public address is:
rsr5WxESewQVf17nzGPnvMvQ2FD6EQYpnt

Here is the relevant log info:
====================================

Configuaring host setup on Evernode...
STAGE Configuring host Xahau account
Using registry: rmv53yu8Wid6kj6AC6NvmiwSXNxRa8vTH
STAGE Configuring Sashimono services
Creating cgroup rules...
Restarting the 'cgrulesengd' service.
Configuring sashimono agent service...
Config file created at /etc/sashimono/sa.cfg
Created symlink /etc/systemd/system/multi-user.target.wants/sashimono-cgcreate.service → /etc/systemd/system/sashimono-cgcreate.service.
Created symlink /etc/systemd/system/multi-user.target.wants/sashimono-agent.service → /etc/systemd/system/sashimono-agent.service.
Installing Evernode Xahau message board...
Updated mb user .bashrc.
STAGE Configuring Xahau message board service
Created symlink /home/sashimbxrpl/.config/systemd/user/default.target.wants/sashimono-mb-xrpl.service → /home/sashimbxrpl/.config/systemd/user/sashimono-mb-xrpl.service.
Installed Evernode Xahau message board.
Registering host on Evernode...
STAGE Registering host on Evernode registry rmv53yu8Wid6kj6AC6NvmiwSXNxRa8vTH
Executing register with params: US 800000 1615040 1677680 37249120 3                 DO-Regular 1 2294 jm.evernode@gmail.com _
Registering host...
Burning unsold URITokens related to the previous leases.
No initiated transfers were found.
Transaction result: tesSUCCESS
Waiting for the sell offer CBF8F841B7FAE4173D0F3C39A6C5F92162A6DB153B4F0FDA2425E1553A17B2FC
Accepting the sell offer..
Transaction result: tesSUCCESS
STAGE Creating lease offers for instance slots...
Offer lease NFT creation error.
Evernode Xahau message board exiting with error.
REG_FAILURE
Rolling back sashimono installation.

**Comment by ejla81** [comment]
I've had exactly same issue

**Comment by ejla81** [comment]
To get access to your secret key at least it's:
sudo nano /home/sashimbxrpl/.evernode-host/.host-account-secret.key 
That's where I am at at the mo, think I'm off to try and reinstall the software on the server

**Comment by ejla81** [comment]
![20240120_113105](https://github.com/EvernodeXRPL/evernode-host/assets/157213308/ede717a9-ae4e-47d1-a22e-14075ccb62f6)
When reinstalling the software on the vps I get that that the host is already registered

**Comment by ejla81** [comment]
I'm in thanks to xrp podcast video below and also doc with all the links in the description is masdively helpful:

https://youtu.be/tiU-u1XsReM?si=8IMdH_Mk0VYfJFsN

https://docs.google.com/document/u/0/d/1n8MSM4aeLN9lGIIFUceHbsVIMFTcX2uWxs00OQA-gDc/mobilebasic

<img width="570" alt="Screenshot 2024-01-20 140155" src="https://github.com/EvernodeXRPL/evernode-host/assets/157213308/64818854-be51-433a-a261-2accac2e149e">

**Comment by jmos1277** [comment]
Well, I managed to get this resolved.  To fix, I did the following:

1) On the same HOST on which the installation failed, I ran the following TRANSFER command:

`curl -fsSL https://raw.githubusercontent.com/EvernodeXRPL/evernode-resources/main/sashimono/installer/evernode.sh | sudo bash -s transfer`

Note that the end of the command is 'transfer' and NOT install.

2) Then, I ran the installer again and specified "n" when asked if the installation was a fresh installation:

`curl -fsSL https://raw.githubusercontent.com/EvernodeXRPL/evernode-resources/main/sashimono/installer/evernode.sh | sudo bash -s install`

Basically, I ran a transfer from my HOST back to the same HOST.

**Comment by FUNDBaron** [comment]
> Well, I managed to get this resolved. To fix, I did the following:
> 
> 1. On the same HOST on which the installation failed, I ran the following TRANSFER command:
> 
> `curl -fsSL https://raw.githubusercontent.com/EvernodeXRPL/evernode-resources/main/sashimono/installer/evernode.sh | sudo bash -s transfer`
> 
> Note that the end of the command is 'transfer' and NOT install.
> 
> 2. Then, I ran the installer again and specified "n" when asked if the installation was a fresh installation:
> 
> `curl -fsSL https://raw.githubusercontent.com/EvernodeXRPL/evernode-resources/main/sashimono/installer/evernode.sh | sudo bash -s install`
> 
> Basically, I ran a transfer from my HOST back to the same HOST.

Unfortunately, this solves the problem only when 500 EVR have been debited, but the host has been registered and upon further calling, the script installs and "transfer" work. I had such a problem yesterday and I solved this problem using the same method. But there is another problem when the host is not registered and the installation was unsuccessful, but 500 EVR were written off and every time the script is run, the installation or transfer script requires you to top up the account address again with 500 EVR, although before this all transactions were completed and were successful.

**Comment by ay432** [comment]
Thank you @jmos1277 A LOT, you helped me so much!!! I had to add 1 EVR to account to complete the transfer!

Thanks again!

**Comment by ay432** [comment]
I am installing another node, 6 slots, deposited 300 XAH and 520 EVR, transfer failed. I'll try to add more XAH. Any other suggestions?

**Comment by ay432** [comment]
Do not forget to resize swap if your VM different from 2 GB.
And installation through transfer seems normal at this point.

**Comment by gadget78** [comment]
> Well, I managed to get this resolved. To fix, I did the following:
> 
> 1. On the same HOST on which the installation failed, I ran the following TRANSFER command:
> 
> `curl -fsSL https://raw.githubusercontent.com/EvernodeXRPL/evernode-resources/main/sashimono/installer/evernode.sh | sudo bash -s transfer`
> 
> Note that the end of the command is 'transfer' and NOT install.
> 
> 2. Then, I ran the installer again and specified "n" when asked if the installation was a fresh installation:
> 
> `curl -fsSL https://raw.githubusercontent.com/EvernodeXRPL/evernode-resources/main/sashimono/installer/evernode.sh | sudo bash -s install`
> 
> Basically, I ran a transfer from my HOST back to the same HOST.

Thankyou for this quick step by step, this has helped many in discord too... 
just to add to this,
IF the above doesn't work, it could be due the install failing BEFORE it accepted the URIToken/NFT.
in which case lets check if its there to accept, or jjust check you have it;

- use https://xahauexplorer.com and sign in with the host address (imported into xaman) once signed in,
select "my NFT offers",
where you can check to see if you can buy your URIToken/NFT (will cost 0.00001 EVR)
will look like this ... 
[image](https://github.com/EvernodeXRPL/evernode-host/assets/35707644/4be61359-70aa-46c0-b098-e18f5480b830)
if none show in this menu, then this is likely not your issue... 
...
but lets confirm that you do own the URIToken/NFT,
select the NFT section, tick the "include NFTs with no media"
that looks like this [image](https://github.com/EvernodeXRPL/evernode-host/assets/35707644/b82bd9b1-0753-4b78-a621-d93e8263dd8e)
it should populate a list, with the last one on the list showing issuer as "Evernode Registry Hook"
and give options to burn or sell, of course DO NOT DO THIS ! 
as you DO own the UriToken/NFT. and this is what cost you 500 EVR!
((screenshooting this part for discord chats can help!)

after carrying out those steps to accept the NFT, or just to confirm it, 
lets go over the next/original steps again.. 
1. TRANSFER
to carry out a transfer, do command `evernode transfer`
FYI, this also fixes the "account already registered"
if that command doesn't work,
try `curl -fsSL https://raw.githubusercontent.com/EvernodeXRPL/evernode-resources/main/sashimono/installer/evernode.sh | sudo bash -s transfer`
-you *may* need to have 1EVR on the host to complete the install,
-and it may ask for the secret path, if it asks, it actually requires the path AND file name, 
which by default is `/home/sashimbxrpl/.evernode-host/.host-account-secret.key`
2. INSTALL
once transferred, you can then retry the install sequence, selecting N, to the "is this a fresh install" question.
`curl -fsSL https://raw.githubusercontent.com/EvernodeXRPL/evernode-resources/main/sashimono/installer/evernode.sh | sudo bash -s install`
just like the "original fix" post above. :)

also to note, althou this fixes the issue to allow the re-install,
it may fail AGAIN like it did before, as the underlying install fault is still present, so you will have to start again.
one way to 100% get a clean install, is to install and use your own xahau node
info on that here https://docs.xahau.network/infrastructure/peering/connect-to-xahau-mainnet

the DEVS know the issue, which is a issue with the install script,
where it doesn't have full error failback code for a terQUEUED error.
so when there's high load on the network, instead of trying again, it just fails..

_also just a FYI...
you can use details in this github post, https://github.com/EvernodeXRPL/evernode-host/issues/144#issuecomment-1898322923 to get and accept the NFT, bbut i moved it down here as its was confusing people !
download the zip file, extract, `cd` into the extracted folder `claim-reg-token`, run `node index.js <address> <secret>`
this will do the same action as the other abbove (accepting the NFT that is waiting for your account) but from the CLI, and not needing to import your secret into xaman etc
also FYI this method just accepts any NFT's, with this method you cannot check if you already own it like the other option_

**Comment by mgjansen63** [comment]
> I'm in thanks to xrp podcast video below and also doc with all the links in the description is masdively helpful:
> 
> https://youtu.be/tiU-u1XsReM?si=8IMdH_Mk0VYfJFsN
> 
> https://docs.google.com/document/u/0/d/1n8MSM4aeLN9lGIIFUceHbsVIMFTcX2uWxs00OQA-gDc/mobilebasic
> 
> <img alt="Screenshot 2024-01-20 140155" width="570" src="https://private-user-images.githubusercontent.com/157213308/298293334-64818854-be51-433a-a261-2accac2e149e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU4NTA3NTAsIm5iZiI6MTcwNTg1MDQ1MCwicGF0aCI6Ii8xNTcyMTMzMDgvMjk4MjkzMzM0LTY0ODE4ODU0LWJlNTEtNDMzYS1hMjYxLTJhY2NhYzJlMTQ5ZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEyMVQxNTIwNTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00OTYzOTJmNTk3MjAwZGJkZjQyYTk0NTk3NjFmZjNhZDJmMmYyMmUyOWY5NzAxMzFiYmU2Yjg3MmY0MGZjYmVmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Z90zA44_hG3mBVNYiaV4gpqvSMkTzWNmoBp2OjWpqU4">

How did you get around the "already Registered" on a Re-Install

**Comment by mgjansen63** [comment]
> I'm in thanks to xrp podcast video below and also doc with all the links in the description is masdively helpful:
> 
> https://youtu.be/tiU-u1XsReM?si=8IMdH_Mk0VYfJFsN
> 
> https://docs.google.com/document/u/0/d/1n8MSM4aeLN9lGIIFUceHbsVIMFTcX2uWxs00OQA-gDc/mobilebasic
> 
> <img alt="Screenshot 2024-01-20 140155" width="570" src="https://private-user-images.githubusercontent.com/157213308/298293334-64818854-be51-433a-a261-2accac2e149e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU4NTA3NTAsIm5iZiI6MTcwNTg1MDQ1MCwicGF0aCI6Ii8xNTcyMTMzMDgvMjk4MjkzMzM0LTY0ODE4ODU0LWJlNTEtNDMzYS1hMjYxLTJhY2NhYzJlMTQ5ZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEyMVQxNTIwNTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00OTYzOTJmNTk3MjAwZGJkZjQyYTk0NTk3NjFmZjNhZDJmMmYyMmUyOWY5NzAxMzFiYmU2Yjg3MmY0MGZjYmVmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Z90zA44_hG3mBVNYiaV4gpqvSMkTzWNmoBp2OjWpqU4">

How did you get around the Already Registered on the Reinstall

**Comment by ejla81** [comment]
the script:
curl -fsSL https://raw.githubusercontent.com/EvernodeXRPL/evernode-resources/main/sashimono/installer/evernode.sh | sudo bash -s transfer

**Comment by Cryptosenseiii** [comment]
I got to the end and ran into this issue.. Any ideas how to fix it? I sent me XAU and Ever tokens to the requested address. Thought I had done it all right but had an issue. 

There was an error during installation. Please provide the file /tmp/evernode/installer-1705961508.log to Evernode team. Thank you.

**Comment by ejla81** [comment]
Try this 
sudo nano /home/sashimbxrpl/.evernode-host/.host-account-secret.key

**Comment by ejla81** [comment]
That will hopefully allow you to access the wallet, then reinstall software using

curl -fsSL https://raw.githubusercontent.com/EvernodeXRPL/evernode-resources/main/sashimono/installer/evernode.sh | sudo bash -s install

**Comment by ejla81** [comment]
Select not a new intall

**Comment by evernodeau** [comment]
For those who can't find the NFT in their wallet, it may be the NFT is in status of sell, you have to buy it back, look in your wallet and find the NFT ID, insert it into https://xahauexplorer.com/nft/ and then purchase it with the same wallet you used to sell

**Comment by evernodeau** [comment]
more info on above found here --> https://github.com/EvernodeXRPL/evernode-host/issues/144#issuecomment-1898456194

**Comment by Cryptosenseiii** [comment]
I got this part to work but my node is still not up and running.. Any ideas
what I can do next? I got my secret key and all that jazz

**Comment by Cryptosenseiii** [comment]
[image: Screenshot 2024-01-22 at 3.50.19 PM.png] Earlier it asked me to
send 500 EVER and 20 XAU to a wallet a QR code that was on the screen.. Do
we know where that went? Doesn't seem to show up. Or was I not suppose to
send that EVER to that QR code?

**Comment by wigster76** [comment]
Installation error keeps happening, Already tried the transfer command and then new installation command but i still get the same error at the end of the installation. NFT also can not be found on the XAHAUexplorer and i see no option to buy the NFT there.

The installation error that i get just says,

There was an error during installation. Please provide the file /tmp/evernode/installer-1707792708.log to evernode team.

Any suggestion or fix for this issue.

**Comment by wigster76** [comment]
Already paid 18.2 XAU and 500 EVR and also put another 500 EVR in my account but still stuck with the same error

**Comment by wigster76** [comment]
Finally got my node working,
After following all the steps mentioned in the thread and i ended up restarting my VPS too.
